### PR TITLE
Added extra fields in event

### PR DESCRIPTION
**Pull request by IloveJavaa**

### CODE REVIEW

File: trackier-ios-sdk/Classes/TrackierEvent.swift [hunk 1]
```diff
@@ -28,6 +28,8 @@ public class TrackierEvent {
     public var param10: String? = ""
 
     var revenue: Float64? = 0.0
+    public var setDiscount: Float64? = 0.0
+    public var setCouponCode: String? = ""
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierEvent.swift:32]: variable name can not be setSomething, plz fix this @IloveJavaa 

create getter/setters for this

File: trackier-ios-sdk/Classes/TrackierSDK.swift [hunk 2]
```diff
@@ -71,4 +71,12 @@ public class TrackierSDK {
     public static func trackAsOrganic(organic: Bool) {
         shared.instance.organic = organic
     }
+    
+    public static func setUserPhone(userPhone: String){
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDK.swift:75]: please note the space after the ) in the above functions @IloveJavaa 
```suggestion
    public static func setUserPhone(userPhone: String) {
```

File: trackier-ios-sdk/Classes/TrackierSDK.swift [hunk 3]
```diff
@@ -71,4 +71,12 @@ public class TrackierSDK {
...
+        shared.instance.customerPhone = userPhone
+    }
+    
+    public static func setUserName(userName: String){
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDK.swift:79]: same as mentioned above
```suggestion
    public static func setUserName(userName: String) {
```

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: trackier-ios-sdk/Classes/TrackierEvent.swift (hunk 1 above)
Comment (IloveJavaa) [reply trackier-ios-sdk/Classes/TrackierEvent.swift:32]: Changes done

Screenshot 2022-08-08 at 11 22 56 AM

File: trackier-ios-sdk/Classes/TrackierSDK.swift (hunk 2 above)
Comment (IloveJavaa) [reply trackier-ios-sdk/Classes/TrackierSDK.swift:75]: Changes done
Screenshot 2022-08-08 at 11 24 46 AM

File: Example/trackier-ios-sdk/AppDelegate.swift [hunk 4]
```diff
@@ -22,7 +22,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
          * In First argument, you need to pass the Trackier SDK api key
         * In second argument, you need to pass the environment which can be either "development", "production" or "testing". */
         
-        let config = TrackierSDKConfig(appToken: "xxxx-xx-xxx-xxx", env: TrackierSDKConfig.ENV_DEVELOPMENT) //Pass your Trackier sdk api key
+        let config = TrackierSDKConfig(appToken: "0455721b-33c5-4c9f-805e-596d818d910a", env: TrackierSDKConfig.ENV_DEVELOPMENT) //Pass your Trackier sdk api key
```
Comment (Hemant-Mann) [inline Example/trackier-ios-sdk/AppDelegate.swift:25]:
```suggestion
        let config = TrackierSDKConfig(appToken: "xxxx-xx-xxx-xxx", env: TrackierSDKConfig.ENV_DEVELOPMENT) //Pass your Trackier sdk api key
```

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**